pkgs/top-level/all-packages.nix

```diff
@@ -679,10 +674,7 @@ in
 
   gamecube-tools = callPackage ../development/tools/gamecube-tools { };
 
-  gams = callPackage ../tools/misc/gams {
-    licenseFile = config.gams.licenseFile or null;
-    optgamsFile = config.gams.optgamsFile or null;
-  };
+  gams = callPackage ../tools/misc/gams (config.gams or {});
```

Comment:
temporary dependency version overrides (in our normal «update/pin old where needed/investigate» upproach) would become a bit more annoying with this style in `all-packages.nix`…

Reply:
Sure, but I did not invent this style, this is the style that is already used in `all-packages.nix`.

Reply:
… unfortunately. Although it seems that this abbreviation indeed doesn't lead to the expected problem in practice.